r/R/install-arrow.R

```diff
@@ -209,16 +216,22 @@ create_package_with_all_dependencies <- function(dest_file = NULL, source_file =
   on.exit(unlink(untar_dir, recursive = TRUE), add = TRUE)
   utils::untar(source_file, exdir = untar_dir)
   tools_dir <- file.path(untar_dir, "arrow/tools")
-  download_dependencies_sh <- file.path(tools_dir, "cpp/thirdparty/download_dependencies.sh")
+  download_dependencies_sh <- file.path(tools_dir, "download_dependencies_R.sh")
```

Comment:
I'm not seeing this file; did you forget to check it in?

Reply:
Thanks, I had forgotten!